ONBOARDING.md

```diff
@@ -42,4 +43,38 @@ Add the new plugin to the [opensearch-plugins meta](https://github.com/opensearc
+4. For publishing to a new platform (other than ones specified above) a new library needs to be added. (ETA: 2 weeks)
+5. **Release Drafter**: Release drafter is a GitHub Action workflow that drafts a release that may or may not contain the release artifacts. The drafted release acts as a trigger to the Jenkins workflow. It also acts as a staging environment for release artifacts. This is to make sure the build environment remains the same even for release artifacts. [Example](https://github.com/opensearch-project/opensearch-py/blob/main/.github/workflows/release-drafter.yml)
+5. **Jenkins Workflow:** Once the Jenkins workflow is added to the repository, onboard the workflow to public CI system https://build.ci.opensearch.org/
+    1. Create a "New Item"
+    2. Name it "\<component-name\>-release"
+    3. Select "Pipeline" as type of the project
+    4. Hit "Ok" and scroll to the bottom of the page
+    5. Select "Pipeline script from SCM" under Pipeline section
+      i. SCM: Github repository link. eg: https://github.com/opensearch-project/opensearch-build
+      ii. Script Path: Relative path to jenkins file. eg: jenkins/check-for-build.jenkinsfile
+    6. Run the workflow once in order to update the configuration of the Jenkins Workflow. You can abort once the workflow starts pulling the docker image. 
+6. **GitHub Webhook**: Add webhook to your GitHub repository by going to repository settings → Webhooks:
+    1. Payload URL: https://build.ci.opensearch.org/generic-webhook-trigger/invoke?token=\<tokenAddedInStep2\>
+    2. Content type: application/json
+    3. SSL verification: Enable
+    4. Which events would you like to trigger this webhook? Select `Releases` only. Please ensure to deselect `Push`
+    5. Active: Enable
+7. Once a webhook is added, it will send a ping to check the connectivity (✅)
+8. Add `RELEASING.md` file to the repository documenting how to release the artifact. [Example](https://github.com/opensearch-project/opensearch-py-ml/blob/main/RELEASING.md)
+9. **Adding tests:** Each library has a respective library tester associated with it that can be used to test you jenkins workflow. This tests can be used to verify that the workflow is making the calls. The build system used is gradle. For example, this [PublishToNpm test](https://github.com/opensearch-project/opensearch-build-libraries/blob/main/tests/jenkins/TestPublishToNpm.groovy) uses [PublishToNpmLibTester](https://github.com/opensearch-project/opensearch-build-libraries/blob/main/tests/jenkins/lib-testers/PublishToNpmLibTester.groovy) with expected parameter that can be unique to your workflow. The assertions makes sure that calls to npm registry is made which is mandatory to release an artifact.
+
```

Comment:
Overall, the on-boarding guide looks good. Can you also add an optional step for 2PR approval process ( if any repo owner is interested in using it)?

Reply:
Thanks for reminding. Let me add that with an example.

Reply:
Added with an example. Thanks!